### ISSUE TITLE
Q: How to handle more than 8 options?

### ISSUE THREAD

**Issue by objectx** [JordanMarr/FSharp.SystemCommandLine#11]
I need help to handle more than 8 options.  The official document suggests [this](https://learn.microsoft.com/en-us/dotnet/standard/commandline/model-binding#parameter-binding-more-than-8-options-and-arguments). But not applicable to `FSharp.SystemCommandLine`.

I've found the following problems:
 * Using `InvocationContext` directly
   * At least, [`AddOption`](https://learn.microsoft.com/en-us/dotnet/api/system.commandline.command.addoption) must be called to register the option, but there is no way to do this.
 * Use [binding](https://learn.microsoft.com/en-us/dotnet/standard/commandline/model-binding#use-a-custom-binder)
   * This requires passing the `IValueDescritptor` to the `SetHandler` call for more than 8 options.<br />( https://github.com/JordanMarr/FSharp.SystemCommandLine/issues/9 is not usable for this case)

Any suggestions?

**Comment by JordanMarr** [comment]
What do you think about this syntax as a way to manually pull parameter values?

```F#
module Program

open FSharp.SystemCommandLine

module Parameters = 
    let words = Input.Option<string[]>(["--word"; "-w"], Array.empty, "A list of words to be appended")
    let separator = Input.OptionMaybe<string>(["--separator"; "-s"], "A character that will separate the joined words.")

let app (ctx: System.CommandLine.Invocation.InvocationContext) =
    // Manually parse parameters
    let words = Parameters.words.GetValue ctx
    let separator = Parameters.separator.GetValue ctx

    // Append words together
    let separator = separator |> Option.defaultValue ", "
    System.String.Join(separator, words) |> printfn "Result: %s"
    0
    
[<EntryPoint>]
let main argv = 
    let ctx = Input.Context()

    rootCommand argv {
        description "Appends words together"
        inputs ctx
        setHandler app
        add Parameters.words
        add Parameters.separator
    }

```

**Comment by JordanMarr** [comment]
This has been implemented in v0.14.0-beta4: 
https://github.com/JordanMarr/FSharp.SystemCommandLine/releases/tag/v0.14.0-beta4

**Comment by objectx** [comment]
@JordanMarr v0.14.0-beta4 works as expected! THANKS! :bow:
